packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -196,8 +196,7 @@ const int kMinColumnWidth = 10;
 ///          [arguments]
 /// ```
 ///
-/// If [columnWidth] is not specified, then the column width will be the
-/// [outputPreferences.wrapColumn], which is set with the --wrap-column option.
+/// [columnWidth] as well as [shouldWrap] have to be provided.
```

Comment:
This comment does not really add much value. These parameters are already marked as required.